### ISSUE TITLE
no-member raised on __getattribute__ for modules

### ISSUE THREAD

**Issue by quazgar** [PyCQA/pylint#6094]
### Bug description

In the following code snippet, pylint complains about line 3 with `E1101: Module 'sys' has no '__getattribute__' member (no-member)`:

```
import sys

print(sys.__getattribute__("argv"))
print(sys.__dict__)

s = "hello"
print(s.__getattribute__("capitalize")())
```

Note that this seems to be OK for other usages of `__getattribute__`.

### Command used

```shell
`pylint test.py`
```

### Pylint output

```shell
************* Module caosadvancedtools.models.test
test.py:1:0: C0114: Missing module docstring (missing-module-docstring)
test.py:3:6: E1101: Module 'sys' has no '__getattribute__' member (no-member)
test.py:6:0: C0103: Constant name "s" doesn't conform to UPPER_CASE naming style (invalid-name)

--------------------------------------------------------------------
Your code has been rated at -4.00/10 (previous run: -4.00/10, +0.00)
```
```


### Expected behavior

Line 3 should be just as OK as line 7.

### Pylint version

```shell
pylint 2.12.2
astroid 2.9.3
Python 3.9.2 (default, Feb 28 2021, 17:03:44)
[GCC 10.2.1 20210110]
```

### OS / Environment

Debian 11
